private-templates-service/client/src/components/AdaptiveCard/AdaptiveCard.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+import * as AdaptiveCards from "adaptivecards";
+import { Card, Container, TemplateName } from "../AdaptiveCard/styled";
+
+function getCard() {
```

Comment:
As you're using Typescript, if the function returns something, you need to specify the _type_ of that return.

Reply:
Done.